api/api/create_pit.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ */
+
+'use strict';
+
+/* eslint camelcase: 0 */
+/* eslint no-unused-vars: 0 */
+
+const { handleError, snakeCaseKeys, normalizeArguments, kConfigurationError } = require('../utils');
+const acceptedQuerystring = [
+  'allow_partial_pit_creation',
+  'keep_alive',
+  'preference',
+  'routing',
+  'pretty',
+  'human',
+  'error_trace',
+  'source',
+  'filter_path',
+];
+const snakeCase = {
+  allowPartialPitCreation: 'allow_partial_pit_creation',
+  keepAlive: 'keep_alive',
+  errorTrace: 'error_trace',
+  filterPath: 'filter_path',
+};
+
+/**
+ * Creates a point in time.
+ * <br/> See Also: {@link https://opensearch.org/docs/latest/opensearch/point-in-time-api#create-a-pit|Opensearch - Create a PIT}
+ * @memberOf API-PIT
+ *
+ * @param {Object} params
+ * @param {string} params.index - The name(s) of the target index(es) for the PIT. May contain a comma-separated list or a wildcard index pattern.
```

Comment:
since the api supports multiple indices , why are we not using 'indices' ?

Reply:
As discussed, to align with the convention used, we are going with `index`